src/_locales/zh/messages.json

```diff
@@ -266,6 +266,7 @@
     "wallets": "錢包",
     "untitled_wallet": "未命名錢包",
     "your_message": "您的留言……",
+    "input_length_exceed_prompt": "{{name}}的長度不應超過{{length}}。",
```

Comment:
```suggestion
    "input_length_exceed_prompt": "{{name}}的長度不能超過{{length}}。",
```

@neruthes @Tedko

Reply:
Both is fine

Reply:
Waiting for @neruthes review the copywriting?